src/test/java/com/booleanuk/core/BasketTest.java

```diff
@@ -4,5 +4,44 @@
 import org.junit.jupiter.api.Test;
 
 class BasketTest {
+    @Test
+    void checkBasketMethods() {
+        Basket basket = new Basket();
```

Comment:
You should split the Assertions into separate tests either for each method of the class or for each entry in the Domain Model, this will make things easier to maintain etc.